examples/flex.rs

```diff
@@ -291,29 +319,16 @@ impl Widget for Example {
                 Constraint::Fixed(_) => Color::Indexed(6),
                 Constraint::Proportional(_) => Color::Indexed(7),
             };
-            self.illustration(*constraint, text, fg).render(*block, buf);
+            let [block, _] = block.split(&Layout::vertical([
+                Fixed(ILLUSTRATION_HEIGHT),
+                Fixed(SPACER_HEIGHT),
+            ]));
+            self.illustration(*constraint, text, fg).render(block, buf);
```

Comment:
Moving the text and fg calculation into illustration (and just passing constraint and area.width) makes this a bit neater.